graphql/src/parser/lexer.rs

```diff
@@ -263,8 +268,8 @@ mod test {
     fn test_query() {
         let input = r"{
           human(id: 1002) {
-            name,
-            appearsIn,
+            name
+            appearsIn
```

Comment:
This change seems unnecessary (it is valid before and after, and since we are testing below without commas, lets keep testing here with commas)

Reply:
@nrc done.